src/program.js

```diff
@@ -617,9 +617,12 @@ Example: $0 --help run.
         type: 'boolean',
       },
       'watch-file': {
-        describe: 'Reload the extension only when the contents of this' +
-                  ' file changes. This is useful if you use a custom' +
-                  ' build process for your extension',
+        alias: ['watch-files'],
+        describe: 'Reload the extension only when the content of these' +
+                  ' files change. This is useful if you use a custom' +
+                  ' build process for your extension or if you want web-ext' +
```

Comment:
Using a "custom build process" is the most common reason for being interested into watching a specific file (or set of files) for changes, and that didn't change with the addition of the alias.
I think that in the end we may just leave the description as is.

Reply:
Got it. Going back...

(PS: started a review by accident)